scss/_variables.scss

```diff
@@ -6,10 +6,10 @@ $blue-light: lighten($blue, 20%);
 $blue-active-highlight: lighten($blue, 10%);
 
 // Grays
-$gray: #899199;
+$gray: #999999;
```

Comment:
`#899199` against a white background wasn't AA-compliant to begin with, so we can't make it even lighter in the Light theme.

I did a [quick contrast check](https://github.com/Automattic/simplenote-electron/issues/795#issuecomment-442735104) the other day, and to meet AA it looks like we won't be able to share the same blue and gray values across both Light/Dark themes.

Reply:
Yeah! I've updated it to `#808080`, which is AA-compliant for the white theme. The same color in the dark theme should now adjust it to be a bit lighter where it is used to `#999999` (#808080 isn't AA-compliant against the #1E1E1E bg)

I also snuck in a fix for the markdown preview with `inline code` syntax, it was changing the font size and looked a bit odd.

Reply:
Dark theme — The grays used for `.dialog-tabs-button` and `.settings-group p` text need to be lightened. (Currently at contrast ratio 2.95)

Light theme — I'm not sure where you got `#808080` as AA-compliant for normal text against `#FFF`, since it is at contrast ratio 3.95. But I'm ok with this value for this PR, since it isn't worse than the current state and we need to make other tweaks to the Light theme colors anyway 👍

Reply:
Interesting! It guess it's ok for Cocoa apps because macOS has an "Increase contrast" option in Accessibility settings...?